src/everest/config/everest_config.py

```diff
@@ -361,6 +396,26 @@ def validate_install_templates_unique_output_files(
         )
         return install_templates
 
+    @model_validator(mode="before")
+    @classmethod
+    def validate_no_data_file(cls, values: dict[str, Any]) -> dict[str, Any]:
+        data_file = values.get("model", {}).get("data_file", None)
+        eclbase = values.get("definitions", {}).get("eclbase", None)
```

Comment:
Should have a different default than None here, so that the text below makes more sense, for example: `<name_of_smspec>`